### ISSUE TITLE
More compatability mods?

### ISSUE THREAD

**Issue by ChaoticNeko** [Lanboost/TModLoader_LansToggleableBuffs#5]
are you planning on doing compatibility for more mods like thorium or soa or just calling it good at calamity

**Comment by Lanboost** [comment]
Hum, I cannot remember correctly, as it was quite a while ago I worked on it, but does it not now automatically find all buffs from all mods?

(if I recall correctly I deprecated the Calamity version of the mod because of this reason)

Otherwise to answer as I no longer play TModLoader I have no real way to test/fix this, and as TModLoader have breaking changes in the new version I have no real interest in updating before they release 1.4 (which imo they should already have done).

If I get an other spike of bordom I might consider it, but as of now I am sad to say no. I will leave as an open issue to see if I can get some spare time.

Otherwise I enjoy that you have enjoyed the old version.
